rules/apps.libsonnet

```diff
@@ -269,6 +294,42 @@
               workload_type: 'job',
             },
           },
+          // workload aggregation for barepods
+          {
+            record: 'namespace_workload_pod:kube_pod_owner:relabel',
+            expr: |||
+              max by(%(clusterLabel)s, namespace, workload, pod) (
+                label_replace(
+                  max by(cluster, namespace, pod, asserts_env, asserts_site) (
+                    last_over_time(
```

Comment:
1. Don't use `last_over_time` here, as the recording rule is evaluated every minute.
2. Don't use Grafana-specific variables (such as `$__range`) as they don't exist in Prometheus.
3. Don't use `asserts_env` and `asserts_site` labels as they're from a separate commercial product